properties/basic.py

```diff
@@ -712,6 +713,8 @@ class String(Property):
 
     * **unicode** - if True, coerce strings to unicode. Default is True
       to ensure consistent behaviour across Python 2/3.
+
+    * **regexp** - regular expression input string must comply with.
```

Comment:
Hmm, looks like both are common. Wikipedia lists them in the order "regular expression, regex or regexp" (`regex` first). This [website](http://www.regular-expressions.info/tutorial.html) makes the case that it is easier to pronounce the plural of `regex`... Sounds like a good enough reason to switch to me. Plus it's shorter.